README.md

```diff
@@ -29,6 +29,20 @@ To keep up to date on what's going on with CoreWCF, you can subscribe to the [an
 * The [Samples](src/Samples/) folder has examples for .NET Framework, .NET Core 3.1, .NET 5 & 6 
 * The [Blog](https://corewcf.github.io/) has details on the design philosophy and a deep dive into the features are included in each new release.
 
+### Use Core WCF project templates (dotnet CLI or VisualStudio)
+
```

Comment:
There was some ambiguity with the name where it was sometimes referred to as "Core WCF" and other times "CoreWCF". We settled on "CoreWCF" and the rest of the documentation has been updated to reflect that. Can you fix the few references here?